azure-pipelines.yml

```diff
@@ -67,7 +67,7 @@ pool:
 
 steps:
 - checkout: self
-  clean: boolean
+  clean: true
```

Comment:
I did this in my PR and it didnt solve the problem.

Reply:
Thanks @white238 -- I didn't see that you already tried.

The next step might be to wrap our ``steps`` task under a ``jobs`` task, and add a ``clean:output`` to the ``job``.

E.g.:

```
jobs:
- job: MyJob
  displayName: My First Job
  continueOnError: true
  workspace:
    clean: outputs                         # <-- This
  steps:
  - script: echo My first job
```
https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=example%2Cparameter-schema#job

Reply:
I also added a make clean in my repo that dropped the build directory from 220megs to 9.. and that made no difference.